git-repository/tests/fixtures/make_am_repo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu -o pipefail
+
+git init -q
+
+echo 1.main > 1
```

Comment:
The issue I am having with naming files with numbers while redirecting into them is that `… >1` has a different meaning than `> 1`, and it's just a little too close for me to feel comfortable while reading the script.

I hope you can see the point and avoid such names in the future, and I am happy to rename it for you for now as well.

Reply:
Ah, the joys of sh.  The only thing I'd add is that for this script I saved the path to the patch in [`${patch_path}`](https://github.com/Byron/gitoxide/blob/e7b2d8f446b41b26b518abf7d1b048605ef2bbe8/git-repository/tests/fixtures/make_am_repo.sh#L14) but the call to [`git am`](https://github.com/Byron/gitoxide/blob/e7b2d8f446b41b26b518abf7d1b048605ef2bbe8/git-repository/tests/fixtures/make_am_repo.sh#L22) references the file directly.

The latter should be `git am "${patch_path}"`.

Reply:
Thanks for the hint. It's adjusted in cfaf31fc654472acf1aacacb516b58a3295cffcd.